docs/storefront/graphql/examples/products.mdx

```diff
@@ -87,6 +87,27 @@ Query all products by not including an argument for `products`.
 
 You can also query for featured products, related products, and more. See the [GraphQL Storefront Playground](https://developer.bigcommerce.com/graphql-storefront/playground) for full schema documentation.
 
+### Get a product by URL using a route node
+
+Query a list of products by matching the `Redirect` type. 
+
+```graphql filename="Get a product with the redirect type" showLineNumbers copy
+
+query {
+  site {
+    route(path: "/laura-ashley-duck-egg-paint/113699-master/", redirect: [FOLLOW|IGNORE] = IGNORE) {
+      redirect # <--- new redirect node of a type Redirect
```

Comment:
It does not seem relevant to include the `redirect` node in this response since the example is specific to getting a product. If you do include it though then it should be mentioned that `redirect` will only exist if the `path` supplied as input points at a redirection. Otherwise it will be `null`.

Reply:
@bc-jz I am a little confused about this ticket. I am confused whether we should document the LookUpUrl capability, the redirect node, or the `route` and `node` nodes.  The title of the ticket says LookUpUrl to account for redirects. The GraphQL example in the ticket has a redirect node. Is this example querying a product or a redirect? Do we need a different example? Do we need two queries, one to query the redirect and then a second one to query a product. I really don't know how this works from just reading the tickets and looking at this example.